src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -535,34 +535,49 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             Return MakeEndOfLineTrivia(GetNextChar)
         End Function
 
+        Private Function TryGet(ByRef Ch As Char) As Boolean
```

Comment:
>TryGet [](start = 25, length = 6)

I'd be tempted to implement this TryGet using the other overload, as `TryGet(0, ByRef ch)`

Reply:
Done